enterprise/backend/src/metabase_enterprise/audit_db.clj

```diff
@@ -227,4 +227,5 @@
   (u/prog1 (maybe-install-audit-db)
    (let [audit-db (t2/select-one :model/Database :is_audit true)]
        ;; prevent sync while loading
-     ((sync-util/with-duplicate-ops-prevented :sync-database audit-db (partial maybe-load-analytics-content! audit-db))))))
+     ((sync-util/with-duplicate-ops-prevented :sync-database audit-db
+        (fn [] (maybe-load-analytics-content! audit-db)))))))
```

Comment:
Using partial when we don't pass in any args felt weird to me, I think this reads a little better.